TLM/TLM/State/OptionsTabs/OptionsGeneralTab.cs

```diff
@@ -424,6 +432,10 @@ public static class OptionsGeneralTab {
             GlobalConfig.WriteConfig();
         }
 
+        private static void OnDifferentiateDefaultSpeedsInNormalViewChanged(bool newVal) {
+            // ?
```

Comment:
Do we need to do anything (eg. invalidate camera/overlay caches) when the option changes?

Reply:
I would suggest maybe selecting theme at the start of drawing and storing in DrawEnv (in `ShowSigns_GUI`).

* A theme for defaults.
* A theme for signs (already exists in `DrawEnv::largeSignsTextures_`)

Then using this selected theme in stead of `RoadSignThemes.Instance.RoadDefaults` (search in code)
No other changes are necessary.

Reply:
Yup - I assume doing that way we would only need to determine what theme to use for defaults only once per render, rather than once per sign icon (many times per render)?

Would there be merit to renaming `largeSignsTextures_` to something less ambiguous, eg. `themedSignTextures_` (and add new `unthemedSignTextures_` too)?

Reply:
Whatever you deem necessary. I tried my best at non-ambiguous names, but if you have a better idea, go ahead and rename